app/model/User.scala

```diff
@@ -59,7 +59,7 @@ object User {
         "addressLine3" -> user.address.line3,
         "addressLine4" -> user.address.line4,
         "postcode" -> user.address.postcode,
-        "country" -> user.address.country,
+        "country" -> user.address.country.map(_.name),
```

Comment:
I stopped myself from just returning the code here to make the transition easier, but maybe now is the time to define the api to use standard codes instead of a particular wording of the names of countries in english. We would probably have to deal with problems with users expecting the countries in the old format though.
@kelvin-chappell  what do you think? should we just return codes instead ?

Reply:
Yes that seems reasonable to me

Reply:
done